recipes/jbig/Makefile.patch

```diff
@@ -0,0 +1,11 @@
+--- Makefile	2014-03-27 14:47:15.000000000 -0400
++++ Makefile	2015-11-06 14:14:48.000000000 -0500
+@@ -4,7 +4,7 @@
+ CC = gcc
+ 
+ # Options for the compiler: A high optimization level is suggested
+-CFLAGS = -O2 -W -Wno-unused-result
++CFLAGS = -O2 -W -fPIC
```

Comment:
Yeah. Those were in the [original recipe](https://github.com/conda/conda-recipes/tree/0dabe789218ef58df023154dfc6482791b336451/jbig). I am not sure we need that patch though. I just removed it and let's see what happens.